### PR TITLE
MGDAPI-4637 - prow: 4.11 for integreatly and cloud resource operator

### PR DESCRIPTION
**Pull request by KevFan**
# Description
Use Openshift 4.11 in prow jobs for integreatly and cloud resource operator master branch

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@KevFan could me move all active feature branch configs to 4.11 as well please.

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-1595-cluster-scoped-3scale.yaml [hunk 1]
```diff
@@ -101,10 +101,14 @@ tests:
           cpu: 100m
     workflow: generic-claim
 - as: multitenant-rhoam-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```
Comment (Boomatang) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-1595-cluster-scoped-3scale.yaml:110]: Was this not meant to be added in a different PR. Or is it a case that the branch needs a rebasing

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-3551-autoscaling.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 base_images:
-  openshift_release_golang-1.17:
+  openshift_release_golang-1.18:
     name: release
     namespace: openshift
-    tag: golang-1.17
+    tag: golang-1.18
```
Comment (Boomatang) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-3551-autoscaling.yaml:5]: This change will be reverted by the ci-operator. If you want to bump the version of go you need to update the openshift-ci/Dockerfile.tools in the integreatly-operator for this branch. Once that change is made these version will be updated.

**Review by Boomatang — CHANGES_REQUESTED** [review]
In my comments I have one general question over the cluster pools and change requested as the change will be undone by the ci-operator. 

It is so nice to see a clean up of the older versions. :)

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-1595-cluster-scoped-3scale.yaml (hunk 1 above)
Comment (KevFan) [reply ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-1595-cluster-scoped-3scale.yaml:110]: Yeah, the previous PR was only for master. Branches would need to be updated also if they want to use cluster pools also, so I thought may aswell keep them consistent

**Comment by KevFan** [comment]
Some failing jobs are due to feature branch requires rebase with master, so I think it should be still fine to merge if these are failing

/retest

**Comment by Boomatang** [comment]
Agreed as they are the rehearse I think it is ok to merge this now. /lgtm

**Comment by KevFan** [comment]
Rebased with master due to conflicts

**Comment by laurafitzgerald** [comment]
/approve
/lgtm
It's fine for some of the rehearsals to fail. those need to be addressed in the repos themselves.
